src/rules/no-default-export.js

```diff
@@ -0,0 +1,40 @@
+module.exports = {
+  meta: {
+    docs: {},
```

Comment:
Can you put a `description` and category in here?

Reply:
I didn't include it because it seemed that none of the other rules appeared to use it ([example](https://github.com/benmosher/eslint-plugin-import/blob/master/src/rules/no-named-default.js)).